docs/Demos.md

```diff
@@ -7,6 +7,196 @@ title: "React-admin demos"
 
 If you want to see what react-admin is capable of, or if you want to learn from apps built by seasoned react-admin developers, check out these demos.
 
+## Overview
+
+Discover our demos that show you the react-admin potential and help you familiarize with it.
+
+<style>
+    .demos-list {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+        grid-gap: 10px;
+    }
+
+    @media (max-width: 600px) {
+        .demos-list {
+            display: flex;
+            flex-wrap: wrap;
+            gap: 10px;
+        }
+    }
```

Comment:
Why switch to `flex` on mobile mode? Why not keep `grid` and only change `grid-template-columns` to be `repeat(1, 1fr);`?